CPAC/pipeline/cpac_pipeline.py

```diff
@@ -1386,13 +1387,16 @@ def build_workflow(subject_id, sub_dict, cfg, pipeline_name=None,
             pipeline_blocks += [alff_falff]
 
     if 'Template' in target_space_alff:
+        if not nuisance_template and not rpool.check_rpool(
+                'space-template_desc-denoisedNofilt_bold'):
```

Comment:
Why doesn't `rpool` have `space-template_desc-denoisedNofilt_bold` if we're running https://github.com/FCP-INDI/C-PAC/blob/4cc96f35874cce8d5e6175029be0b1a913b439a2/CPAC/nuisance/nuisance.py#L2592-L2621 ?

Reply:
I think this just wasn't added when it should have been - this is needed here.

Reply:
What I was confused about here is how I was getting C-PAC complaining about not having `space-template_desc-denoisedNofilt_bold` in the resource pool after

https://github.com/FCP-INDI/C-PAC/blob/d1d06a53c77bdb5848b666b882bff0298394100f/CPAC/pipeline/cpac_pipeline.py#L1347-L1349

before I added `and not rpool.check_rpool('space-template_desc-denoisedNofilt_bold'):` to the condition in https://github.com/FCP-INDI/C-PAC/blob/d1d06a53c77bdb5848b666b882bff0298394100f/CPAC/pipeline/cpac_pipeline.py#L1389-L1392

but maybe I just made a mistake testing it. Either way, it seems to work okay with this `if` block in place.